content/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst

```diff
@@ -0,0 +1,230 @@
+===============
+Cluster picking
+===============
+
+.. _inventory/misc/cluster_picking:
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+.. |SOS| replace:: :abbr:`SOs (Sales Orders)`
+
+Cluster picking is an advanced order fulfillment approached derived from :ref:`batch picking
+<inventory/misc/batch_picking>`. In this strategy, pickers load a cart with multiple packages, each
+designated for specific a *sales order* (SO). Then, the picker travels to each storage location and
+places the products directly in the package related to the customer order.
+
+This method is most efficient for medium-sized companies with high order volumes relatively few
+unique products, since the method eliminates the need for sorting products into packages for
+customers after picking. Disadvantages of cluster picking are that urgent orders cannot be
+prioritized and optimized batches must be manually created beforehand, so the picking process can
+lead to bottlenecks.
+
+.. _inventory/misc/cluster_picking/example:
+
+.. example::
+   #. |SO| 1 calls for one apple and orange
+   #. |SO| 2 requires one apple and banana
+   #. |SO| 3 needs one apple, orange, and banana
+
+   Apples are stored in Shelf A, oranges in Shelf B, and bananas in Shelf C.
+
+   To pick products for three orders at once, the cart is loaded with three empty packages.
+
+   Starting at Shelf A, the the picker places apples into each package. Next, the picker navigates
+   to Shelf B and places oranges in the packages designated for |SO| 1 and |SO| 3. Finally, the
+   picker pushes the cart to Shelf C and loads packages for |SO| 2 and |SO| 3 with a banana, each.
+
+   With all three |SOS| fulfilled, the picker pushes the cart to the output location, where
```

Comment:
This is a bit of a nitpick, but I would say that sales orders are considered "fulfilled" when they're shipped to the customer.
```suggestion
   With the packages for all three |SOS| packed, the picker pushes the cart to the output location, where
```